### ISSUE TITLE
Missing Az.EventGrid cmdlets

### ISSUE THREAD

**Issue by aelij** [Azure/azure-powershell#19299]
### Description

The cmdlets `New-AzEventGridSystemTopic` and `New-AzEventGridSystemTopicEventSubscription` are missing from the `Az.EventGrid` module, even though they have published documentation.

I have downloaded the [nupkg](https://www.powershellgallery.com/packages/Az.EventGrid/1.3.0) from the gallery and opened it in ILSpy, and indeed these cmdlets are missing. However they are available in Cloud Shell (same module version).

### Issue script & Debug output

```PowerShell
> New-AzEventGridSystemTopic
New-AzEventGridSystemTopic: The term 'New-AzEventGridSystemTopic' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
```

### Environment data

```PowerShell
Name                           Value
----                           -----
PSVersion                      7.2.6
PSEdition                      Core
GitCommitId                    7.2.6
OS                             Microsoft Windows 10.0.22000
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

### Module versions

```PowerShell
ModuleType Version    PreRelease Name                                ExportedCommands
---------- -------    ---------- ----                                ----------------
Script     2.9.1                 Az.Accounts                         {Add-AzEnvironment, Clear-AzConfig, Clear-AzContext, Clear-AzDefault…}
Script     1.3.0                 Az.EventGrid                        {Get-AzEventGridDomain, Get-AzEventGridDomainKey, Get-AzEventGridDomainTopic, Get-AzEventGridSubscription…}
```

**Comment by dingmeng-xue** [comment]
Thanks for reporting. It is changelog was not updated when introducing new cmdlet. It has been fixed by https://github.com/Azure/azure-powershell/pull/19151

**Comment by aelij** [comment]
@dingmeng-xue the problem is not the changelog, it's that the assembly in the package feed (version 1.3.0) does not contain those types.

**Comment by aelij** [comment]
Got it, thanks :) So how come Cloud Shell was updated? Built on a different pipeline?

**Comment by dingmeng-xue** [comment]
Cloud Shell installs Az by default. If version of module is greater than or equal to 1.0, module is included in Az. Then user can get the latest module once Cloud Shell finishes deployment. If version of module is lower than 1.0, user needs to install it manually.

**Comment by dingmeng-xue** [comment]
This issue will be fixed tomorrow when Az 8.3 release

**Comment by jaehwanjoa** [comment]
2022-11-04: 
New-AzEventGridSystemTopic: The term 'New-AzEventGridSystemTopic' is not recognized as a name of a cmdlet, function, script file, or executable program.
Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
